classes/Hook.php

```diff
@@ -1244,4 +1250,21 @@ private static function getMethodName(string $hookName): string
     {
         return 'hook' . ucfirst($hookName);
     }
+
+    /**
+     * Return status from a given hook name.
+     *
+     * @param string $hook_name Hook name
+     *
+     * @return bool
+     */
+    public static function getHookStatusByName($hook_name): bool
+    {
+        $sql = new DbQuery();
+        $sql->select('active');
+        $sql->from('hook', 'h');
+        $sql->where('h.name = "' . $hook_name . '"');
```

Comment:
```suggestion
        $sql->where('h.name = "' . pSQL($hook_name) . '"');
```

Reply:
Don't forget to escape queries in order to prevent SQL injections

Reply:
Sorry I forgot, had already done it here https://github.com/PrestaShop/PrestaShop/pull/20849/commits/c376c9bb52485488e56b9e51e45c6d559ec130bd. Thanks !